k8s/view-api/stage-view-api.yaml

```diff
@@ -26,63 +26,64 @@ spec:
         - name: view-api
           image: us.gcr.io/airqo-250220/airqo-stage-view-api:latest
           imagePullPolicy: Always
-          resources: {}
-            # requests:
-            #   memory: 200Mi
-            #   cpu: "100m"
-            # limits:
-            #   memory: 700Mi
-            #   cpu: "1000m"
+          resources:
+            requests:
+              cpu: 100m
+              memory: 250Mi
+            limits:
+              cpu: 200m
+              memory: 700Mi
           ports:
             - containerPort: 8080
               name: view-api
           envFrom:
             - configMapRef:
                 name: stage-view-api-config
-          readinessProbe:
-            initialDelaySeconds: 30
-            httpGet:
-              port: 8080
-              path: api/v1/view/actuator/health/readiness
-          livenessProbe:
-            initialDelaySeconds: 30
-            httpGet:
-              port: 8080
-              path: api/v1/view/actuator/health/liveness
+          # readinessProbe:
+          #   initialDelaySeconds: 30
+          #   httpGet:
+          #     port: 8080
+          #     path: api/v1/view/actuator/health/readiness
+          # livenessProbe:
+          #   initialDelaySeconds: 30
+          #   httpGet:
+          #     port: 8080
+          #     path: api/v1/view/actuator/health/liveness
```

Comment:
Hi @123MwanjeMike any reason for removing the readiness and liveness probes??

Reply:
They were stopping the pods from starting. I guess they were not picking the right information. Removing them seemed to solve the issue. Perhaps you could double check to be sure.

Reply:
https://platform.airqo.net/api/v1/view/actuator/health/liveness
https://platform.airqo.net/api/v1/view/actuator/health/readiness

We need to put them back. They are very useful when performing updates and many other usecases. If the pod is not starting, then something is not right and needs to be fixed. If we remove them, we shall never know that the pod has issues.

https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/
https://developers.redhat.com/blog/2020/11/10/you-probably-need-liveness-and-readiness-probes
https://blog.colinbreck.com/kubernetes-liveness-and-readiness-probes-how-to-avoid-shooting-yourself-in-the-foot/

Reply:
Thanks @NoahNsimbe 
I've added the probes back